clarity-ext-scripts/clarity_ext_scripts/covid/rtpcr_analysis_service.py

```diff
@@ -51,7 +51,7 @@ class RTPCRAnalysisService(object):
 
     def __init__(self, covid_reporter_key, internal_control_reporter_key):
         self._covid_reporter_key = covid_reporter_key
-        self._internal_control_reporter_key = internal_control_reporter_key
```

Comment:
I think for consistency it would be nice to expose both of these as "public". But we can do that later.